### ISSUE TITLE
Validation error if choice as object in list

### ISSUE THREAD

**Issue by minidron** [bhch/django-jsonform#92]
Array validation support only flat choices. If use as object, get error "Invalid choice". [django_jsonform/validators.py](https://github.com/bhch/django-jsonform/blob/6266cf2b86fcccd896d19d2066c6e0ff1c027d79/django_jsonform/validators.py#L138)
```
{
  'type': 'list',
  'items': {
    'type': 'integer',
    'choices': [{
      'title': 'one',
      'value': 1
    }]
  }
}
```
Invalid choice

```
{
  'type': 'list',
  'items': {
    'type': 'integer',
    'choices': [1]
  }
}
```
No errors

**Comment by bhch** [comment]
Thanks for reporting this. I'll release a fix soon.

**Comment by bhch** [comment]
The fix has been released in [v2.16.0](https://github.com/bhch/django-jsonform/releases/tag/v2.16.0). Please update. 

Thank you for reporting this bug.
